sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -743,6 +743,32 @@ def pcollection_count_monitoring_infos(self, tag_to_pcollection_id):
     return infos
 
 
+class SdfTruncateSizedRestrictions(DoOperation):
+  def __init__(self, *args, **kwargs):
+    super(SdfTruncateSizedRestrictions, self).__init__(*args, **kwargs)
+    self.sdf_process_op = None
+    self.consumers_count = 0
+
+  def current_element_progress(self):
+    # type: () -> Optional[iobase.RestrictionProgress]
+    if self.consumers_count == 1 and self.sdf_process_op is None:
+      return None
+    return self.sdf_process_op.current_element_progress()
+
+  def try_split(self, fraction_of_remainder):  # type: (...) -> Optional[Any]
```

Comment:
Please fix the type, after #12275 it is:
Optional[Tuple[Iterable[SdfSplitResultsPrimary], Iterable[SdfSplitResultsResidual]]